services/observer/tokens/tokens.go

```diff
@@ -0,0 +1,91 @@
+// Package tokens populates the address <=> token mappings cache.
+//
+// TODO Is this package in the right spot?
```

Comment:
@EnoRage Please check. Might not make sense to put this under the observer package.